### PR TITLE
A slabinfo fix, and the v1 (1.1.0) release

### PR DESCRIPTION
**Pull request by brenns10**
The slabinfo fix was more involved than I wanted, but it works well, is compatible with drgn 0.0.25 - 0.0.27, and it actually makes the `slabinfo` module modestly faster.

### CODE REVIEW

File: buildrpm/python-drgn-tools.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2024, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
 Name:           python-drgn-tools
-Version:        0.9.1
+Version:        1.0.0
```
Comment (biger410) [inline buildrpm/python-drgn-tools.spec:4]: Here is the main branch, so it is still 1.0.0? You will create a new branch for v1 and set version to 1.1.0 there and also change the version here to 2.0.0, right?

Comment (brenns10) [reply buildrpm/python-drgn-tools.spec]: No, I totally missed that. This should go to 1.1.0, I'll create the `stable/v1` branch, and then on the main branch I'll bump to 2.0.0.

File: buildrpm/python-drgn-tools.spec [hunk 2]
```diff
@@ -61,6 +61,9 @@ rm %{buildroot}/usr/bin/DRGN
 %{_mandir}/man1/corelens.1.gz
 
 %changelog
+* Tue Aug 27 2024 Stephen Brennan <stephen.s.brennan@oracle.com> - 1.1.0-1
```
Comment (biger410) [inline buildrpm/python-drgn-tools.spec:64]: should this be 1.1.0 instead of 1.1.0-1?

Comment (brenns10) [reply buildrpm/python-drgn-tools.spec:64]: No, most RPMs have a release version appended with the hyphen, it starts at 1. If we update the packaging alone, we'll update that number.

Comment (brenns10) [reply buildrpm/python-drgn-tools.spec:64]: It comes from line 5 above.

**Review by biger410 — APPROVED** [review]
Looks good.

**Pull request merged**